Makefile

```diff
@@ -257,7 +266,7 @@ image-push-scorecard-test-kuttl-multiarch:
 
 test: test-unit ## Run the tests
 
-test-sanity: tidy build/operator-sdk lint
+test-sanity: tidy build/operator-sdk
```

Comment:
Did you mean to remove it from test-sanity? I'm not sure what others do but I usually run `make test-sanity` before submitting my PRs to make sure everything's okay.

Reply:
If we keep it will be tested twice in the cI unnecessarily. 
I think it is ok run `make lint` and `make  test-sanity` WDYT?

Reply:
Another option might be n the Travis todo the make-sanity calls but not use the target.